src/asset/entities.rs

```diff
@@ -54,12 +55,22 @@ impl PyLoadedProp {
     fn color(&self) -> [f32; 4] {
         self.color
     }
+
+    fn properties(&mut self) -> BTreeMap<String, String> {
```

Comment:
I have all the methods exposed by the rust code also declared in the plumber.pyi file, so that vscode can detect them. Could you also add this there? Also applies to the methods added to PyLight etc.